opentok/opentok.py

```diff
@@ -414,11 +424,17 @@ def create_session(
 
     def headers(self):
         """For internal use."""
+        user_agent = (
+            "OpenTok-Python-SDK/" + __version__ + " python/" + platform.python_version()
```

Comment:
```suggestion
            "OpenTok-Python-SDK/" + self._app_version+ " python/" + platform.python_version()
```

`app_version` passed in via the constructor overrides the actual version of the SDK, so that should stay here as well (not perfect, but it's the current interface)